src/test/java/org/cactoos/list/BehavesAsList.java

```diff
@@ -64,12 +64,12 @@ public boolean matchesSafely(final List<E> list) {
         new Assertion<>(
             "must have an index for the sample item",
             list.indexOf(this.sample),
-            new Verifies<>(i -> i >= 0)
+            new Satisfies<>(i -> i >= 0)
```

Comment:
@victornoel `new IsGreaterOrEqualThan(0)` ?

Reply:
@andreoss I don't think this exists, do you have something particular in mind?